models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -656,46 +662,30 @@ aeif_cond_beta_multisynapse_dynamics( double,
   // good compiler will optimize the verbosity away ...
 
   // shorthand for state variables
-  const double& V = y[ S::V_M ];
+  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bound V
```

Comment:
done. However when I will commit please check if this is sufficient or please suggest a more suitable comment.